database/src/main/java/info/metadude/android/eventfahrplan/database/repositories/LecturesDatabaseRepository.kt

```diff
@@ -82,8 +82,7 @@ class LecturesDatabaseRepository(
             return lectures.toList()
         }
 
-        cursor.moveToFirst()
-        while (!cursor.isAfterLast) {
+        while (cursor.moveToNext()) {
```

Comment:
Please also apply this pattern to the other repositories to have a consistent code base.

Reply:
I reverted this change and will create a new pull request for that pattern.